manager/src/main/resources/application.properties

```diff
@@ -26,4 +26,5 @@ quarkus.flyway.create-schemas=true
 %dev.quarkus.datasource.username=event-connect
 %dev.quarkus.datasource.password=event-connect
 %dev.quarkus.hibernate-orm.log.sql=true
-%dev.quarkus.hibernate-orm.log.bind-parameters=true
\ No newline at end of file
+%dev.quarkus.hibernate-orm.log.bind-parameters=true
+%dev.quarkus.hibernate-orm.database.generation=drop-and-create
```

Comment:
We should move this to use Flyway. We can copy the setup and approach that we had from DaaS, apart from using PostGres rather than H2.

There is a section in the DaaS Fleet Manager README.md that I think we can use for our approach to Flyway: https://github.com/kiegroup/baaas-decision-fleet-manager#database-development

WDYT?

Reply:
Actually I tried to remove this to let flyway create the schemas but it did not work out. Do you have any suggestion? I've also looked at the baaas stuff but the configuration is the same (except for this line)

Reply:
Is there any log from Flyway at all when Quarkus starts?

Reply:
@robpblake 
```
2021-09-01 11:14:21,909 INFO  [org.fly.cor.int.lic.VersionPrinter] (Quarkus Main Thread) Flyway Community Edition 7.11.2 by Redgate
2021-09-01 11:14:21,910 INFO  [org.fly.cor.int.dat.bas.BaseDatabaseType] (Quarkus Main Thread) Database: jdbc:postgresql://localhost:5432/event-connect (PostgreSQL 13.1)
2021-09-01 11:14:21,929 INFO  [org.fly.cor.int.com.DbValidate] (Quarkus Main Thread) Successfully validated 0 migrations (execution time 00:00.007s)
2021-09-01 11:14:21,929 WARN  [org.fly.cor.int.com.DbValidate] (Quarkus Main Thread) No migrations found. Are your locations set up correctly?
2021-09-01 11:14:21,938 INFO  [org.fly.cor.int.com.DbMigrate] (Quarkus Main Thread) Current version of schema "public": << Empty Schema >>
2021-09-01 11:14:21,941 INFO  [org.fly.cor.int.com.DbMigrate] (Quarkus Main Thread) Schema "public" is up to date. No migration necessary.
```

Reply:
Ah wait... I see that in baaas we have the manual schema creation...